### PR TITLE
Bug 1539182: Detect if ClusterResourceOverrides enabled during console install

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1539182

/assign @sdodson 
/cc @jwforres @jupierce

### CODE REVIEW

File: roles/openshift_web_console/tasks/install.yml [hunk 1]
```diff
@@ -71,6 +71,9 @@
     - set_fact:
         config_to_migrate: "{{ master_config_output.content | b64decode | from_yaml }}"
 
+    - set_fact:
+        cro_plugin_enabled: "{{ config_to_migrate.admissionConfig is defined and config_to_migrate.admissionConfig.pluginConfig is defined and config_to_migrate.admissionConfig.pluginConfig.ClusterResourceOverrides is defined }}"
```
Comment (spadgett) [inline roles/openshift_web_console/tasks/install.yml]: @sdodson Let me know if there's a better way to check if a nested key is defined.

Comment (sdodson) [reply roles/openshift_web_console/tasks/install.yml]: I think we'd considered writing a filter that would handle this more cleanly but I'm not finding such a filter. So this is pretty common across the code base.

**Comment by sdodson** [comment]
/lgtm
We'll need to revisit this in the future as we go further down the road towards making components more isolated without access to shared objects like master config but during the transition this is fine.

**Comment by sdodson** [comment]
bot, retest this please

**Comment by sdodson** [comment]
I think the fedora/27/atomic test is failing because it's using a version of openshift that crashes whenever there's no assetConfig.

**Pull request merged**